### PR TITLE
Added support for multiple regions and added per-site r^2 optional output

### PR DESCRIPTION
**Pull request by saulpierotti**
I added the optional output that is produced when using the flag --out_r2_per_site and modified the code for producing input.txt so that it can also be run in multiple chromosomes by providing a list of regions as input in val(region). This still allows for a single input.

Running on multiple regions is important to calculate aggregated MAF bin metrics for the whole genome for example.

I also removed the indentation from the version HEREDOCS since it is causing problems with dumpsoftwareversions.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/glimpse2/concordance/main.nf [hunk 1]
```diff
@@ -2,19 +2,21 @@
 
 nextflow.enable.dsl = 2
```
Comment (LouisLeNezet) [inline tests/modules/nf-core/glimpse2/concordance/main.nf:3]: Could you add one test with a list for the region ?

**Review by LouisLeNezet — COMMENTED** [review]
Nice edit !
How do you specify the region as a list ?

**Review by LouisLeNezet — APPROVED** [review]
Sorry for the delay !
Everything is fine for me :)

**Comment by saulpierotti** [comment]
> Sorry for the delay ! Everything is fine for me :)

No worries! :)

**Pull request merged**